src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesTransportFactory.php

```diff
@@ -44,7 +44,13 @@ public function create(Dsn $dsn): TransportInterface
         }
 
         if ('ses+smtp' === $scheme || 'ses+smtps' === $scheme) {
-            return new SesSmtpTransport($user, $password, $region, $this->dispatcher, $this->logger);
+            $transport = new SesSmtpTransport($user, $password, $region, $this->dispatcher, $this->logger);
+
+            if (null !== ($pingThreshold = $dsn->getOption('ping_threshold'))) {
```

Comment:
the brackets after `!==` should be removed

Reply:
Removed, ty